packages/echo-base/src/__tests__/errors/NetworkError.test.ts

```diff
@@ -41,3 +51,43 @@ describe('NetworkError', () => {
         expect(ne.message).not.toEqual(ne.name);
     });
 });
+
+describe('NetworkError & subclasses', () => {
+    const message = 'Network Error Testing';
+    const httpStatusCode = 500;
+    const url = 'http://localhost:3000';
+    const exception = { NetworkException: 'endpoint is unreachable' };
+
+    const networkArgs = { message, httpStatusCode, url, exception };
+    const expectedProperties = { httpStatusCode, url, ...exception };
+
+    test.each([
+        ['NetworkError', new NetworkError(networkArgs)],
+        ['BadRequestError', new BadRequestError(networkArgs)],
+        ['ForbiddenError', new ForbiddenError(networkArgs)],
+        ['UnauthorizedError', new UnauthorizedError(networkArgs)],
+        ['NotFoundError', new NotFoundError(networkArgs)],
+        ['ValidationError', new ValidationError(networkArgs)],
+        ['BackendError', new BackendError(networkArgs)]
+    ])('%p should preserve default name, message and properties', (firstArg, secondArg) => {
+        const error = secondArg as BaseError;
+        expect(error.name).toBe(firstArg);
+        expect(error.message).toBe(message);
+        expect(error.getProperties()).toStrictEqual(expectedProperties);
+    });
+
+    test.each([
```

Comment:
Ah, I didn't know that `test` was an alias for `it`!

I was thinking about maybe just using `it` everywhere, to be consistent...but in this case `test` makes it more readable. 🤔